autils/archive/archive.py

```
@@ -0,0 +1,539 @@
+
+
+def probe_zstd_cmd():
+    """
+    Attempts to find a suitable zstd tool that behaves as expected
+
+    :rtype: str or None
+    :returns: path to a suitable zstd executable or None if not found
+    """
+    zstd_cmd = shutil.which("zstd")
+    if zstd_cmd is not None:
+        proc = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)